checkov/terraform/checks/resource/gcp/GoogleStoragePublicAccessPrevention.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+class GoogleStoragePublicAccessPrevention(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Public access prevention should be enforced"
+        id = "CKV_GCP_112"
```

Comment:
```suggestion
        name = "Ensure public access prevention is enforced on Cloud Storage bucket"
        id = "CKV_GCP_114"
```
we already have 2 other new checks before yours, so this would be next available ID 🙂

Reply:
yeah, sadly we have nothing to track ongoing IDs, if they are currently in an open PR.